model/config.go

```diff
@@ -752,6 +753,18 @@ func (o *Config) SetDefaults() {
 		*o.EmailSettings.EmailBatchingInterval = EMAIL_BATCHING_INTERVAL
 	}
 
+	if o.EmailSettings.EnableSMTPAuth == nil {
+		o.EmailSettings.EnableSMTPAuth = new(bool)
+		if o.EmailSettings.ConnectionSecurity == CONN_SECURITY_NONE {
+			*o.EmailSettings.EnableSMTPAuth = false
+		} else {
+			*o.EmailSettings.EnableSMTPAuth = true
+		}
+		if o.EmailSettings.ConnectionSecurity == CONN_SECURITY_PLAIN {
```

Comment:
I don't think I see an issue with doing this. In the rare case that the "EnableSMTPAuth" setting  has been implemented, but somehow this is still set to "Plain" this would catch that case as well.